judoscale-shoryuken/rails-autoscale-shoryuken.gemspec

```diff
@@ -0,0 +1,30 @@
+lib = File.expand_path("../lib", __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require "judoscale/shoryuken/version"
+
+Gem::Specification.new do |spec|
+  spec.name = "rails-autoscale-shoryuken"
+  spec.version = Judoscale::Shoryuken::VERSION
+  spec.authors = ["Adam McCrea", "Carlos Antonio da Silva", "Jon Sullivan"]
+  spec.email = ["hello@judoscale.com"]
+
+  spec.summary = "This gem provides Shoryuken integration with the Judoscale autoscaling add-on for Heroku."
```

Comment:
We're no longer just "for Heroku", but I know this is the same summary we have for the other gems. I think I want to change all of their summaries to something short and sweet like "Autoscaling for [job backend] workers". Judoscale is already in the gem name, so I don't think we even need to mention it in the summary.

Do you mind sneaking that change in here or creating a separate PR for it?

Reply:
Pushed separately just to make it easier to review: https://github.com/judoscale/judoscale-ruby/pull/206